py-polars/pypolars/frame.py

```diff
@@ -100,6 +100,28 @@ def read_csv(
         n_threads: Optional[int] = None,
         dtype: "Optional[Dict[str, DataType]]" = None,
     ) -> "DataFrame":
+        """
+        Read a comma-seperated value file into a Dataframe.
+
+        Parameters
+        ---
+        - file `str`, `TextIO`  
+            Any valid filepath can be used. Example: `file.csv`.
+        - sep `str`  
+            Default is `,`.
+        - stop_after_n_rows `int`   
+            Only read specified number of rows of the dataset. After `n` stops reading. Default is `None`. 
+        - has_headers `bool`  
+            Indicate if first row of dataset is header or not. If set to False first row will be set to `column_x`, `x` being an enumeration over every column in the dataset. Default = `True`.
+        - encoding `str`  
+            Default = `utf8`.
```

Comment:
Maybe show the allowed encodings:

* "utf8"
*  "utf8-lossy"

Where lossy means that invalid utf8 values are replaced with � character.